v1.1/set-vars.md

```diff
@@ -109,26 +110,10 @@ The following demonstrates how to assign a list of values:
 (1 row)
 ~~~
 
-## `SET TIME ZONE`
```

Comment:
@knz, is there any reason we can't apply these `time zone` changes to v1.0 docs as well?